### PR TITLE
WIP: frontend pdmp meds display, first pass

### PR DESCRIPTION
**Pull request by achen2401**
frontend changes after hitting PDMP meds end point.
Not quite ready.  Just a first pass so others here can have a look.

### CODE REVIEW

**Comment by ivan-c** [comment]
[Launch URL for this Pull Request](https://launch.smarthealthit.org/launcher?launch_uri=https%3A%2F%2Fdeploy-preview-5--distracted-jepsen-bc9c39.netlify.com%2Flaunch.html&patient=5c41cecf-cf81-434f-9da7-e24e5a99dbc2&fhir_ver=4)

File: src/components/summary.json [hunk 1]
```diff
@@ -609,5 +609,44 @@
...
+        {
+          "headers": {
+            "Drug": {
+              "key": "medicationCodeableConcept",
+              "formatter": "codeableConceptFormat",
+              "formatterArguments": ["medicationCodeableConcept", "text"],
+              "minWidth": 200
+            },
+            "Written": { "key": "dateWritten", "formatter": "datishFormat" },
+            "NIC Code": {
```
Comment (ivan-c) [inline src/components/summary.json]: I think this should be NDC (National Drug Code); NIC (eGov) is the vendor providing the PDMP service
```suggestion
            "NDC Code": {
```

Comment (ivan-c) [reply src/components/summary.json]: ....I guess *NDC code* is redundant then :)

Comment (achen2401) [reply src/components/summary.json]: thank you Ivan for catching that. should be fixed now.

File: src/components/Landing.js [hunk 2]
```diff
@@ -59,6 +70,36 @@ export default class Landing extends Component {
...
+   */
+  async getExternalData() {
+      const externalDatasetKey = 'ExternalDataSet';
+      let dataSet = {};
+      dataSet[externalDatasetKey] = {};
+      //PDMP data
+      //loading local file - this is temporary, just to demo rendering
+      //let response = await fetch(`${process.env.PUBLIC_URL}/pdmp.json`, {mode: 'no-cors'})
+      /*
+       * IF environmental variable REACT_APP_CONF_API_URL is set, call that endpoint
```
Comment (ivan-c) [inline src/components/Landing.js]: Can you remove this code if you don't think you'll use it, or maybe make conditional on `REACT_APP_CONF_API_URL`?

File: src/components/Summary.js [hunk 3]
```diff
@@ -328,22 +342,31 @@ export default class Summary extends Component {
...
-                {this.renderSection("HistoricalTreatments")}
               </Collapsible>
-
-            {/*  <Collapsible trigger={this.renderSectionHeader("RiskConsiderations")} open={true}>
+			
+              <Collapsible trigger={this.renderSectionHeader("RiskConsiderations")} open={true}>
                 {this.renderSection("RiskConsiderations")}
-              </Collapsible>
-			 */} 
+              </Collapsible> */}
```
Comment (ivan-c) [inline src/components/Summary.js]:
```suggestion
              </Collapsible> }
```

**Pull request merged**